File: src/CadastroAgentes.Business/Models/Cliente.cs
using System;

namespace CadastroAgentes.Business.Models
{
    public class Cliente : Entity
    {
        public string Nome { get; set; }
        public string Endereco { get; set; }
        public DateTime DataNascimento { get; set; }
        public float Altura { get; set; }
        public float Peso { get; set; }

        public Status Status { get; set; }
        public Guid? UsuarioAnalise { get; set; }
        public DateTime? DataTerminoAnalise { get; set; }
    }
}

File: src/CadastroAgentes-App/src/app/_models/Cliente.ts
import { Status } from './Status';

export class Cliente {
    Id: string;
    Nome: string;
    Endereco: string;
    DataNascimento: Date;
    Altura: number;
    Peso: number;
    Status?: Status;
    UsuarioAnalise?: string;
    DataTerminoAnalise?: Date;
}
